services/emailService.py

```
@@ -0,0 +1,30 @@
+    msg.set_content(body)
+
+    # Attach the CSV file
+    msg.add_attachment(csv_bytes, maintype="text", subtype="csv", filename=filename)
+
+    try:
+        context = ssl.create_default_context()
+        with smtplib.SMTP_SSL(smtp_server, port, context=context) as server:
+            server.login(sender_email, app_password)
+            server.send_message(msg)
+        return True
+    except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)